### ISSUE TITLE
jLocation doesn't work if GPS was not enabled at startup

### ISSUE THREAD

**Issue by r1me** [jmpessoa/lazandroidmodulewizard#28]
If GPS was initially (before starting the app) disabled, then `jLocation` will never call back with phone location nor GPS status / satellites.

How to reproduce with `AppLocationDemo1` demo:
1. Close apps, disable GPS.
2. Start `AppLocationDemo1` (don't press any button)
3. Enable GPS in phone Settings.
4. Press 'Try Start Tracker' button in demo app.
5. The app will return a message 'GPS is On! Starting Tracker...' but will never call any of the `jLocation` events.

Android 7.1.2

**Comment by r1me** [comment]
Fixed. 
Once `jlocation` is created, it tries to get a best provider but ONLY from a list of currently enabled. If GPS is disabled at startup, it is not considered as a location provider, unless `jLocation` object is recreated. This is because of a call to [getBestProvider](https://developer.android.com/reference/android/location/LocationManager.html#getBestProvider%28android.location.Criteria,%20boolean%29) with a second parameter `enabledOnly` set to true. I changed:
````
mProvider = mLocationManager.getBestProvider(mCriteria, true);
````
to
````
mProvider = mLocationManager.getBestProvider(mCriteria, false);
````
in `java\lamwdesigner\jLocation.java`.
